src/scripts/cfo.zig

```diff
@@ -92,15 +102,24 @@ const Fuzzer = enum {
     }
 
     fn spawn_command(fuzzer: Fuzzer, shell: *Shell, seed: u64) !std.ChildProcess {
-        if (fuzzer == .vopr or fuzzer == .vopr_testing) {
-            const state_machine: []const []const u8 =
-                if (fuzzer == .vopr) &.{} else &.{"-Dsimulator-state-machine=testing"};
+        if (fuzzer == .vopr or fuzzer == .vopr_testing or
+            fuzzer == .vopr_lite or fuzzer == .vopr_testing_lite)
+        {
+            const state_machine: []const []const u8 = if (fuzzer == .vopr or fuzzer == .vopr_lite)
+                &.{}
+            else
+                &.{"-Dsimulator-state-machine=testing"};
+            const lite: []const []const u8 = if (fuzzer == .vopr or fuzzer == .vopr_testing)
+                &.{}
+            else
+                &.{"--lite"};
```

Comment:
Why `.{"string"}` instead of `"string"`?

Reply:
This interpolates not into a string, but into an array of CLI arguments. The shell syntax supports splats: in `"foo {bar}"`, `bar` can be either a string, or a list of strings, and it'll do the right thing in both cases. This gives raise to the idiom of optional arguments `bar` can be a singleton or an empty list.

We could also add explicit support for optionals to shell, so that you can intrepolate `?[]const u8`.

Somewhat orthogonally, but perhaps we should split `{bar}` sytnax into `{bar}`, `{bar...}` and `{bar?}` to sintactically signify the artity of interpolation (the arity is "obvious" from the types, but an extra mark could perhaps make the code a bit more readable)

Reply:
Again, hope to refactor this whole thing away!

Reply:
I fully admit that this particular part of the file is write-only :sob: